### PR TITLE
feat: add new strategies

### PR DESCRIPTION
**Pull request by DarkGhost7**

### CODE REVIEW

**Comment by DarkGhost7** [comment]
Ready for review this is all outstanding strategies that are missing

**Review by jstashh — APPROVED** [review]

**Review by milkyklim — CHANGES_REQUESTED** [review]
Need a small update for Solidex Joint Provider

File: data/strategies/250/SolidexSingleSidedStable.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "strategy",
+  "name": "Solidex Stable Yield Seeker",
+  "description": "Supplies {{token}} to a [Solidex Finance](https://solidexfinance.com/#/pools) Stable Pool to earn SOLID and SEX. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
+  "addresses": [
+    "0xADE3BaC94177295329474aAd6A253Bae979BFA68",
```
Comment (milkyklim) [inline data/strategies/250/SolidexSingleSidedStable.json:6]: Seems like duplicate strategy.

File: data/strategies/250/SolidexSingleSidedVolatile.json [hunk 2]
```diff
@@ -2,7 +2,10 @@
   "$schema": "strategy",
   "name": "Solidex Volatile Yield Seeker",
   "description": "Supplies {{token}} to a [Solidex Finance](https://solidexfinance.com/#/pools) Volatile Pool to earn SOLID and SEX. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
-  "addresses": ["0xADE3BaC94177295329474aAd6A253Bae979BFA68"],
+  "addresses": [
+    "0xADE3BaC94177295329474aAd6A253Bae979BFA68",
```
Comment (milkyklim) [inline data/strategies/250/SolidexSingleSidedVolatile.json:6]: Seems like duplicate strategy.

Comment (DarkGhost7) [reply data/strategies/250/SolidexSingleSidedVolatile.json:6]: Yes was just fixing

**Pull request merged**